terraform/environments/delius-core/ec2_instance_db.tf

```diff
@@ -79,6 +79,18 @@ data "aws_iam_policy_document" "core_shared_services_bucket_access" {
   }
 }
 
+data "aws_iam_policy_document" "ec2_access_for_ansible" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "ec2:DescribeTags",
+      "ec2:DescribeInstances",
+      "ec2:DescribeVolumes"
+    ]
+    resources = ["*"]
```

Comment:
Im assuming there is no way to narrow this down to just the relevant EC2 instances?

Reply:
Consulting [this](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html), I don't think there's a way to do this. The actions just operate at the List access level